contracts/GroupingWorkflows.sol

```diff
@@ -106,21 +104,23 @@ contract GroupingWorkflows is
     }
 
     /// @notice Mint an NFT from a SPGNFT collection, register it with metadata as an IP, attach
-    /// Programmable IP License Terms to the registered IP, and add it to a group IP.
+    /// license terms to the registered IP, and add it to a group IP.
     /// @dev Caller must have the minter role for the provided SPG NFT.
     /// @param spgNftContract The address of the SPGNFT collection.
     /// @param groupId The ID of the group IP to add the newly registered IP.
     /// @param recipient The address of the recipient of the minted NFT.
-    /// @param licenseTermsId The ID of the registered PIL terms that will be attached to the newly registered IP.
+    /// @param licenseTermsId The ID of the registered PIL terms that will be attached to the new IP.
+    /// @param licenseTemplate The address of the license template used for `licenseTermsId`.
     /// @param ipMetadata OPTIONAL. The desired metadata for the newly minted NFT and registered IP.
     /// @param sigAddToGroup Signature data for addIp to the group IP via the Grouping Module.
     /// @return ipId The ID of the newly registered IP.
     /// @return tokenId The ID of the newly minted NFT.
-    function mintAndRegisterIpAndAttachPILTermsAndAddToGroup(
+    function mintAndRegisterIpAndAttachLicenseAndAddToGroup(
         address spgNftContract,
         address groupId,
         address recipient,
         uint256 licenseTermsId,
+        address licenseTemplate,
```

Comment:
It would be better put licenseTemplate ahead of licenserTermsId in the param list.

Reply:
Resolved in [ca8032b](https://github.com/storyprotocol/protocol-periphery-v1/pull/55/commits/ca8032be0474d59840f8ab0023bf5ee6582413ad) :)